### DISCUSSION TITLE
TownHall Narrative

### DISCUSSION THREAD

**Discussion by j-p-courneya** [Relay-Maryland/relaymaryland.com#12] [Ideas]
## Town Hall

One of many treasures in Historic Relay is our town hall.  This historic building has been around since the late 1800’s and still stands proudly today.  It’s uses have been plenty.  It started out as the first fire department in southwest Baltimore only later to become a multi purpose center for the Relay Women’s Club, theatre & arts center, and library.  More recently, we host our quarterly community meetings welcoming neighbors, friends, and our district representatives from the county and Maryland.    

In an effort to maintain this grand 2-story stature that is pleasing to anyone who stands in it’s presence, we also rent out the hall for private events that exclusively funds the ongoing maintenance that private citizens commit countless hours towards.

Town Hall rentals allow small parties enjoy the quaint beauty of Relay.   Most private parties use our hall for such functions as bridal showers, birthdays, graduations where the guest list does not exceed 60.  The hall is equipped with two bathrooms, full sized kitchen, refrigeration and a parking lot for up to 15 cars.   If you are interested in finding out more about renting the hall, please email us at relaytownhall@gmail.com.

@brianzelip

**Comment by brianzelip** [comment]
Done
